editor/action_map_editor.cpp

```diff
@@ -353,7 +354,7 @@ void ActionMapEditor::_notification(int p_what) {
 	switch (p_what) {
 		case NOTIFICATION_ENTER_TREE:
 		case NOTIFICATION_THEME_CHANGED: {
-			action_list_search->set_right_icon(get_theme_icon(SNAME("Search"), SNAME("EditorIcons")));
+			action_list_search->set_right_icon(get_theme_icon(SNAME("Search"), EditorStringName(EditorIcons)));
```

Comment:
`get_theme_icon(name, SNAME("EditorIcons"))` is so common that I thought a while back we could provide a help method for this, like `get_editor_theme_icon(name)`.

Reply:
Where would be this method located? In the same singleton?

Reply:
I'm thinking directly in Control, so it becomes:
```suggestion
			action_list_search->set_right_icon(get_editor_theme_icon(SNAME("Search")));
```

Reply:
WDYT about this @YuriSizov?

Reply:
Ideally, every control, including editor widgets, should have a manual theme cache. So the only place where these could be called would be the `_update_theme_item_cache` method. Which makes it less of a nuisance.

But in practice we're far from it. I can certainly see the appeal of `get_editor_theme_icon(SNAME("Search"))` over `get_theme_icon(SNAME("Search"), EditorStringName(EditorIcons))`. So that would probably be a good change. Make sure to do it in Window as well. Methods should be only available with TOOLS_ENABLED.

Reply:
This makes "EditorIcons" string name useless .-.